include/bluetooth/gatt.h

```diff
@@ -811,7 +811,7 @@ struct bt_gatt_notify_params {
  *  @param conn Connection object.
  *  @param params Notification parameters.
  *
- *  @return 0 in case of success or negative value in case of error.
+ *  @return request id in case of success or negative value in case of error.
```

Comment:
Can the application safely ignore the request id and keep working as it was prior to the introduction of IDs?

Reply:
It can, it would just not be able to cancel it.